### DISCUSSION TITLE
Fosslight_dependency 2.1.0

### DISCUSSION THREAD

**Discussion by giocol91** [fosslight/fosslight#1089] [Show and tell]
I am experimenting with your tool with all the different languages provided. I am finding some troubles with new .NET project configuration. The tool seems to work only in the old .NET format via the manifest in xml format (packages.config) and not the new json format (under obj/project.assets.json). I can also only walk through the depencies of just 10% of the nuget packages involved, is there any possibility to have the url list you search for and the possibility to pass a token to command line?

**Comment by giocol91** [comment]
[nuget.patch](https://github.com/user-attachments/files/17847213/nuget.patch)
I have solved the problem with the nuget package manager, i'll do a PR, for now this is the patch. This ensures compatibility with NuGet's case-sensitive API otherwise it works only with the lower case packages.
Remain the problem with the manifest format, I have tested also with your test_nuget and confirm that is broken

**Comment by dd-jy** [comment]
@giocol91 
Thanks for contributing the nuget.patch!
I created a PR(https://github.com/fosslight/fosslight_dependency_scanner/pull/245) and merged it so that the content can be applied.

Also fixed an issue in the PR(https://github.com/fosslight/fosslight_dependency_scanner/pull/243) that wasn't analyzed for the new json format (obj/project.assets.json).

Please note that these PRs will be reflected in the v4.1.8 release.
